modular_skyrat/modules/cryosleep/code/cryopod.dm

```diff
@@ -168,6 +172,8 @@ GLOBAL_LIST_EMPTY(ghost_records)
 
 // This is not a good situation
 /obj/machinery/cryopod/Destroy()
+	if(!quiet && src.z != 2)
+		GLOB.station_cryopods -= src
```

Comment:
We don't need to have a check for this, BYOND is actually smart enough to not cause an error when trying to remove something that wasn't present in the list to begin with.
```suggestion
	GLOB.station_cryopods -= src
```